### ISSUE TITLE
2019-11-26 - tc011, tc012, tc071 M1SS - computation engine prints unnecessary output to console

### ISSUE THREAD

**Issue by jhhughes256** [tensfeldt/openNCA#59]
### Issue

When running tc011, tc012 and tc071, additional output is printed to the console than what is desired. This output is not affected by changes in OPTIMIZKEL. This issue does not occur for tc010, tc013, tc014, tc015, tc016, tc017 or tc018.

An example of this output from tc012 is shown below.

```
[1] 24280050
[1] CONC    TIME    INT_EXT
<0 rows> (or 0-length row.names)
[1] 45831434
[1] CONC    TIME    INT_EXT
<0 rows> (or 0-length row.names)
[1] 70745987
[1] CONC    TIME    INT_EXT
<0 rows> (or 0-length row.names)
[1] 72354714
[1] CONC    TIME    INT_EXT
<0 rows> (or 0-length row.names)
[1] 130509330
[1] CONC    TIME    INT_EXT
<0 rows> (or 0-length row.names)
```

**Comment by jhhughes256** [comment]
### Update

As of commit 5af8ad1a75 slightly less output is printed to console, but unnecessary output is still printed. Example below.

```
[1] 24280050
[1] 45831434
[1] 70745987
[1] 72354714
[1] 130509330
[1] 147299665
[1] 246409536
[1] 246953363
[1] 249265153
[1] 283369224
[1] 292748221
[1] 297583299
[1] 328228072
[1] 330747542
```

**Comment by rudraya** [comment]
This issue was addressed in 7fd866a, please let me know if it still occurs.

**Comment by tensfeldt** [comment]
This issue doesn't occur with commit 22d43ea and will be closed.
